### ISSUE TITLE
UIActionInputModule - all PointerEventDatas have their button set to Left regardless of pressed button

### ISSUE THREAD

**Issue by covisa** [Unity-Technologies/InputSystem#477]
All PointerEventDatas received by components implementing IPointerClickHandler, IDragHandler etc. have the button field set to InputButton.Left even when the event originates from Middle Click Action or Right Click Action.

Action setup:
![actions](https://user-images.githubusercontent.com/37138133/54610945-b674cd00-4a5e-11e9-8884-49420b167f89.png)

Test script:
```
using UnityEngine;
using UnityEngine.EventSystems;

public class OnPointerClickTest : MonoBehaviour, IPointerClickHandler {
    public void OnPointerClick(PointerEventData eventData) {
        Debug.Log("OnPointerClick: " + eventData.button);
    }
}
```

Resulting log using all different mouse buttons:
![log](https://user-images.githubusercontent.com/37138133/54611145-17040a00-4a5f-11e9-862e-36794e36d45b.png)

When using StandaloneInputModule the button field is set as expected.

Reproduced on Windows 10 with Unity 2019.1.0b7 and InputSystem 0.2.1 (also affects earlier InputSystem and Unity versions).

**Comment by JiaUnity** [comment]
If I'm not mistaken, the PointerEventData.button uses the old input mouse button. I don't think it works with the new ISX. 
@Rene-Damm can you confirm this?

**Comment by JiaUnity** [comment]
Nevermind. This is a bug if it happens. 
Problem is it doesn't happen on my end:
![Capture](https://user-images.githubusercontent.com/33637761/56071800-970f4e00-5d45-11e9-8330-7aaed910e2bb.JPG)
@covisa have you tried 0.2.6-preview or the latest Develop branch?

**Comment by StayTalm** [comment]
I Repro'd your issue, and found the underlying problem, a fix is incoming.

**Comment by JiaUnity** [comment]
This is fix in the latest Develop (01d708f). Closing.
